docker-compose.yml

```
@@ -0,0 +1,32 @@
+    container_name: interview-be-app
+    restart: always
+    env_file:
+      - .env
+    expose:
+      - 8080
+    healthcheck:
+      test: [ "CMD", "curl", "-fsS", "http://localhost:8080/actuator/health" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 60s
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)